test/test-functions

```diff
@@ -935,11 +934,51 @@ install_debian_systemd() {
     done < <(grep -E '^Package:' "${SOURCE_DIR}/debian/control" | cut -d ':' -f 2)
 }
 
+install_suse_systemd() {
+    local testsdir=/usr/lib/systemd/tests
+    local pkgs
+
+    dinfo "Install SUSE systemd"
+
+    pkgs=(
+        systemd udev
+        systemd-container
+        systemd-coredump
+        systemd-experimental
+        systemd-journal-remote
+        systemd-portable
+    )
+
+    for p in ${pkgs[*]}; do
+        rpm -q $p &>/dev/null || continue
+
+        ddebug "Install files from package $p"
+        for f in $(rpm -ql $p); do
```

Comment:
```bash
while read -r f; do
...
done < <(rpm -ql "$p")
```

Reply:
Again I'm not sure to see the point in changing that, care to explain ?